src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -363,7 +363,7 @@ namespace winrt::TerminalApp::implementation
                 if (!path.empty())
                 {
                     const auto buffer = control.ReadEntireBuffer();
-                    til::io::write_utf8_string_to_file_atomic(std::filesystem::path{ path }, til::u16u8(buffer));
+                    til::io::write_utf8_string_to_file_atomic(std::filesystem::path{ path.c_str() }, til::u16u8(buffer));

```

Comment:
..? this should be `std::wstring_view{ path }`

Reply:
FWIW: in general _stop using `c_str` when you have a smart string type that knows its own length._

Reply:
uhg I forgot that there wasn't an implicit `hstring` to `wstring_view`